org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.java

```diff
@@ -1019,6 +1023,25 @@ protected IResourceServiceProvider getResourceServiceProvider(URI uri) {
 		options.setCancelIndicator(cancelIndicator);
 		return renameService.prepareRename(options);
 	}
+	
+	/**
+	 * @since 2.25
+	 */
+	public CompletableFuture<List<FoldingRange>> foldingRange(FoldingRangeRequestParams params) {
```

Comment:
Here we use a List of FoldingRange - does it make sense to document if we expect multiple folding ranges on the same line?

Reply:
Just saw that we have JavaDoc on the interface. I think it's vague enough and good enough at the same time.